### DISCUSSION TITLE
Limit the number of value on the y axis

### DISCUSSION THREAD

**Discussion by Philippe2705** [patrykandpatrick/vico#434] [Questions] [answered]
Hi all,

I am trying to setup Vico to show a graph, but I can't find any documentation on how to limit the number of value on the y axis.
![image](https://github.com/patrykandpatrick/vico/assets/16638011/7f269bdd-00d4-42a2-87b7-531ecfc3487b)
In this exemple, there is 11 values displayed on the y axis, which is way to much, how can I limit this number?

Also, how can I change the color of the value on the y axis?

Many thanks for the help

Here is my code if required:
```@Composable
fun RegularLineChart(dataPoint: ChartEntryModel) {
    val yellow = Color(0xFFFFAA4A)
    val marker = rememberMarker()
    Chart(
        chart = lineChart(
            lines = listOf(
                lineSpec(
                    lineColor = yellow,
                    lineBackgroundShader = verticalGradient(
                        arrayOf(yellow.copy(0.5f), yellow.copy(alpha = 0f)),
                    ),
                ),
            ),
            axisValuesOverrider = AxisValuesOverrider.adaptiveYValues(yFraction = 1f, round = true),

        ),
        modifier = Modifier.height(280.dp),
        model = dataPoint,
        startAxis = rememberStartAxis(
            axis = LineComponent(
                color = Color.White.toArgb(),
                thicknessDp = 0f,
                shape = Shapes.pillShape,
            ),
        ),
        bottomAxis = rememberBottomAxis(
            tickLength = 0.dp,
            axis = LineComponent(
                color = MaterialTheme.colors.surface.toArgb(),
                thicknessDp = 0f,
                shape = Shapes.pillShape,
            ),
            guideline = LineComponent(
                color = MaterialTheme.colors.surface.toArgb(),
                thicknessDp = 0f,
                shape = Shapes.pillShape,
            ),
            labelRotationDegrees = -30f,
            valueFormatter = { index, _ ->
                LocalDateTime.now().minusDays(index.toLong()).format(DateTimeFormatter.ofPattern("MM-dd"))
            },

        ),
        isZoomEnabled = false,
        marker = marker,

    )
}```

**Comment by patrickmichalik** [comment, marked as answer]
Hello! To limit the label count, you can use `AxisItemPlacer.Vertical.default`, which has a `maxItemCount` parameter. From [the “Axes” page of the wiki](https://patrykandpatrick.com/vico/wiki/axes):

> For [`VerticalAxis`](https://patrykandpatrick.com/vico/api/core/com.patrykandpatrick.vico.core.axis.vertical/-vertical-axis), use [`AxisItemPlacer.Vertical`](https://patrykandpatrick.com/vico/api/core/com.patrykandpatrick.vico.core.axis/-axis-item-placer/-vertical), a base implementation of which can be created via [`AxisItemPlacer.Vertical.default`](https://patrykandpatrick.com/vico/api/core/com.patrykandpatrick.vico.core.axis/-axis-item-placer/-vertical/-companion/default). This function lets you customize the maximum item count.

To change the label color, you can create a custom `ChartStyle` and use `ProvideChartStyle` to provide it or, more simply, pass in a custom `TextComponent`. Here‘s one possible approach:

```kt
rememberStartAxis(label = axisLabelComponent(color = Color.White), ...)
```

**Reply by Philippe2705** [reply]
Thanks a lot! :)

**Reply by patrickmichalik** [reply]
You’re welcome!
